src/hard_sphere/mod.rs

```diff
@@ -3,10 +3,9 @@
 use feos_core::{HelmholtzEnergyDual, StateHD};
 use ndarray::*;
 use num_dual::DualNum;
-use std::borrow::Cow;
+use std::{borrow::Cow, sync::Arc};
```

Comment:
```suggestion
use std::borrow::Cow;
```

Reply:
Ran `cargo fmt` on the whole project which should capture this.